mkdocs.yml

```diff
@@ -93,7 +93,7 @@ nav:
   - Data Dictionary:
       - About: Data_Dictionary/index.md
       - Viewer: Data_Dictionary/viewer.md
-      - Search: Data_Dictionary/gdcmvs.md
+      - Search: https://gdc-mvs.nci.nih.gov/gdc/" target="_blank" rel="noopener noreferrer
```

Comment:
In order to have the link open a new tab,  i'm using html-injection at build-time.

Reply:
The quotes look weird, but based on the pattern of the quotation marks in home.html this seems to be intentional .

Reply:
yeah, this is a html-injection exploit. the unbalanced quotes are intentional. the template looks something like:
\<a href="{url}">

and it'll replace `{url}` with our value.  we can inject html attributes by playing with the quotes:

\<a href="`https://gdc-mvs.nci.nih.gov/gdc/" target="_blank" rel="noopener noreferrer`">